browser/ui/brave_actions/brave_action_view_controller.cc

```diff
@@ -10,6 +10,7 @@
 #include <utility>
 
 #include "brave/browser/ui/brave_actions/brave_action_icon_with_badge_image_source.h"
+#include "brave/browser/profiles/profile_util.h"
```

Comment:
nit: order

Reply:
Thanks, I'll force-push fix & skip CI right before merging.

Reply:
Sorry, I forgot to push this before merging, I'll do a quick follow-up fix on this.